src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -434,13 +444,33 @@ def flow_test(
                     generator_outputs = self._get_generator_outputs(line_result.output)
                     if generator_outputs:
                         logger.info(f"Some streaming outputs in the result, {generator_outputs.keys()}")
+                if allow_generator_output and chat_output_name:
+                    self._raise_error_when_test_failed(line_result, show_trace=True)
+                    print(f"{Fore.YELLOW}Bot: ", end="")
+                    # Since streaming get_result_output need connect with server, it needs flow_executor is live, so
+                    # print chat output here
+                    print_chat_output(line_result.output[chat_output_name], generator_record)
```

Comment:
still can't understand why we need to print chat output here

Reply:
client.stream api won't pass all response one time, if executor destroy when print, will raise connection error

Reply:
client.stream won't pass all response one time, if executor is destroied, it will kill service and connection will closed.

Reply:
offline sync, we may reconsider the lifecycle of executor proxy in interactive/ui chat flow scenario.